test.js

```diff
@@ -1,28 +1,30 @@
 var mocha = require('mocha');
 var chai = require('chai');
+var type = require('type-detect');
 
 var describe = mocha.describe;
 var it = mocha.it;
 var expect = chai.expect;
 
-chai.use(require('./index'));
+var roughly = require('./index');
+chai.use(roughly);
 
-describe('expect(...).to.roughly.deep.equal(...)', function() {
-  it('passes for empty objects', function() {
+describe('expect(...).to.roughly.deep.equal(...)', function () {
+  it('passes for empty objects', function () {
```

Comment:
done, sorry about that - my editor must have reformatted that.  
I also removed a space in `function (` in index.js for consistency.